guides/UPGRADING.md

```diff
@@ -0,0 +1,46 @@
+Technology Upgrading Policy
+=======================
+
+## Overview
+
+Our approach to version upgrades is contingent on the technology in question. Details of which are outlined below, per technology and language.
+
+If you notice any upgrades that are not covered by this policy (including major versions of DBs becoming available), please
+raise them on the 20% board for triage.
+
+We don’t update dependencies in legacy apps unless there is a high or severe security vulnerability present (babbage, zebedee, train, brian,
+legacy florence).
```

Comment:
I'm not sure what legacy florence is? perhaps links to these would be helpful 🙂

Reply:
I've extracted the copy for the legacy apps into its own section now. I've also added a few extra words to clarify what 'legacy florence' is 👍🏽

Reply:
That's much clearer 🙌